dashboard/lib/expired_deleted_account_purger.rb

```diff
@@ -156,14 +157,16 @@ def report_results
     upload_metrics metrics unless @dry_run
   end
 
-  def build_metrics(review_queue_depth)
+  def build_metrics(review_queue_depth, manual_reviews_needed)
     {
       # Number of accounts purged during this run
       AccountsPurged: @num_accounts_purged,
       # Number of accounts queued for manual review during this run
       AccountsQueued: @num_accounts_queued,
       # Depth of review queue after this run (may include auto-retryable entries)
-      ManualReviewQueueDepth: review_queue_depth,
+      ReviewQueueDepth: review_queue_depth,
```

Comment:
Do you know why `QueuedAccountPurge.count` was so high and not decreasing? If these are accounts which should be auto cleaned up, why was the queue never decreasing in size?

Reply:
I don't really know what happened. The queue started to shrink from Jan 10th, maybe related to something you did here?
<img width="440" alt="Screen Shot 2021-01-22 at 10 32 59 AM" src="https://user-images.githubusercontent.com/46507039/105530813-3395ac80-5c9d-11eb-88ea-e8531ba1c855.png">
<img width="266" alt="Screen Shot 2021-01-22 at 10 33 13 AM" src="https://user-images.githubusercontent.com/46507039/105530841-3ee8d800-5c9d-11eb-8756-cbe66b16dee3.png">